docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+
+### Elements of the Foundational Architecture
+
+The foundational architecture is designed with the following fictitious elements:
+
+Domain site `contosowebstore.com`
+
+User roles used to illustrate the use case, and provide insight into the user interface.
+
+#### Role: Site and Subscription Admin
+
+|Item      |Example|
+|----------|------|
+|Username: |`adminXX@contosowebstore.onmicrosoft.com`|
+| Name: |`Global Admin Azure PCI Samples`|
+|User type:| `Subscription Administrator and Azure Active Directory Global Administrator`|
+
+* The admin account cannot read credit card information unmasked. All actions are logged.
+* The admin account cannot manage or log into SQL Database.
+* The admin account can manage Active Directory and subscription.
+
+#### Role: SQL Administrator
+
+|Item      |Example|
+|----------|------|
+|Username: |`sqlAdmin@contosowebstore.onmicrosoft.com`|
+| Name: |`SQLADAdministrator PCI Samples`|
+| First name: |`SQL AD Administrator`|
+|Last name: |`PCI Samples`|
+|User type:| `Administrator`|
+
+* The sqladmin account cannot view unfiltered credit card information. All actions are logged.
+* The sqladmin account can manage SQL database.
+
+#### Role: Clerk
+
+|Item      |Example|
+|----------|------|
+|Username:| `receptionist_EdnaB@contosowebstore.onmicrosoft.com`|
+| Name: |`Edna Benson`|
+| First name:| `Edna`|
+|Last name:| `Benson`|
+| User type: |`Member`|
+
+Edna Benson is the receptionist and business manager. She is responsible for ensuring that customer information is accurate and billing is completed. Edna is the user logged in for all interactions with the Contoso Webstore demo website. Edna has the following rights: 
+
+- Edna can create and read customer information
+- Edna can modify customer information.
+- Edna can overwrite or replace credit card number, expiration, and CVV information.
+
+> In the Contoso Webstore, the user is automatically as the **Edna** user for testing the capabilities of the deployed environment.
+
+### Contoso Webstore - Azure pricing sample calculation
+
+The solution cost sample has a monthly fee structure and a use per hour to
+consider when sizing the solution. This example deployment **estimated** cost using the [Azure costing calculator](https://azure.microsoft.com/en-us/pricing/calculator/). The solution 
+consist of the following items:
+
+| **Service type** | **Custom name** | **Region** | **Description** | **Estimated Cost** | 
+| ----------------- | --------------- | ----------- | -------------- | ----------------- | 
+| Virtual Machines | Virtual Machines | South Central US | 1 Standard virtual machine(s), 1 Standard virtual machine(s), A2 v2 (2 cores, 4 GB RAM, 20 GB disk) size: 744 hours | $101.18 | 
+| App Service | App Service | South Central US | 1 instance(s), 744 hours, size: P1, premium tier, 0 SNI connection(s), 0 IP connection(s) | $223.20 | 
+| IP Addresses | IP Addresses | East US | arm type, 2 public IP Address(es) x 744 hours | $5.95 | 
```

Comment:
Why are the VM and App Service plan deployed to a different region than the rest of the solution?